helperApp/plaid/plaid.js

```diff
@@ -1,25 +1,16 @@
+require('./plaid.scss');
```

Comment:
We can include the css inside the Javascript, webpack does that by default. However it was causing CSP issues, we'd have to tweak those if we want everything inside the js bundle. That might be no bueno for security though.